SharkCage/CageNetwork/MsgService.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <string>
+
+enum class ServiceMessage
+{
+	START_CM = 0,       // Start CageManager
+	STOP_CM,            // Stop CageManager
+	START_PC,           // Start Process (in CageManager)
+	STOP_PC				// Stop Process (in CageManager)
```

Comment:
Pls correct line indenting of the comment

Reply:
this was because we are using tabs and they have different lenghts on different systems (the comments were aligned on my side)